### PR TITLE
Adding the stash measurement functionality to AuthorizeAndStashCmd

### LINKED ISSUE #1660: Add stashing functionality to Manifest based Authorization feature.

### PR DESCRIPTION
**Pull request by mhatrevi**

### CODE REVIEW

**Review by swenson — DISMISSED** [review]
LGTM

File: runtime/src/authorize_and_stash.rs [hunk 1]
```diff
@@ -78,8 +78,18 @@ impl AuthorizeAndStashCmd {
 
             let flags: AuthAndStashFlags = cmd.flags.into();
             if !flags.contains(AuthAndStashFlags::SKIP_STASH) {
-                // TODO: Stash the image hash
-                Err(CaliptraError::RUNTIME_UNIMPLEMENTED_COMMAND)?;
+                // TODO do we need to return this?
```
Comment (jhand2) [inline runtime/src/authorize_and_stash.rs:81]: nit: remove TODO

**Review by jhand2 — CHANGES_REQUESTED** [review]
This is the same as https://github.com/chipsalliance/caliptra-sw/pull/1752. Can we chery-pick that commit instead of creating a new one?

Yes, I mentioned that in the Slack message. This is "cherry-picked". The other PR is abandoned.

File: runtime/src/authorize_and_stash.rs (hunk 1 above)
Comment (mhatrevi) [reply runtime/src/authorize_and_stash.rs:81]: Removed.

**Review by jhand2 — APPROVED** [review]

**Pull request merged**